about_processes.exs

```diff
@@ -0,0 +1,37 @@
+defmodule AboutProcesses do
+  use Koans
+
+  think "You are a process with a process ID" do
+    pid_name = inspect(self)
+
+    assert_? pid_name =~ ~r/#PID<\d+\.\d+\.\d+>/
+  end
+
+  think "Processes receive messages" do
+    send self, {:hello, "world"}
+
+    receive do
+      {:hello, message} -> assert message == __?
+    end
+  end
+
+  think "A spawned process is a function running in parallel" do
+    spawn __?
```

Comment:
Initially I wasn't thrilled that there is no assertion in this example, but I actually do like getting the reader to figure out that they need to spawn a function.